tests/test_image_props/test_standard_rolling_image_props.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import numpy as np
+from waveanalysis.image_props.image_properties import get_multi_frame_properties
+import json
+
+@pytest.fixture
+def default_filepaths():
+    return [
+        '/Users/domchom/Documents/GitHub/ZS_wave_analysis/tests/assets/standard/1_Group1.tif',
```

Comment:
you can change all of these to relative paths (ie delete `/Users/domchom/Documents/GitHub/ZS_wave_analysis/`) since the user should be running `pytest` from the root directory and obviously won't have the upstream file path in their system

Reply:
Probably because `.csv` is in the .gitignore file, oops! you can use `git add -f path/to/your/file.csv` to force add a .csv file